packages/core/CHANGELOG.md

```diff
@@ -5,6 +5,35 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [0.0.11](https://github.com/waku-org/js-waku/compare/core-v0.0.10...core-v0.0.11) (2023-02-11)
+
+
+### ⚠ BREAKING CHANGES
+
+* bump typescript
+* bump all prod dependencies
+* bump libp2p dependencies
+
+### Features
+
+* Codec as a property of the protocol implementations ([a5ff788](https://github.com/waku-org/js-waku/commit/a5ff788eed419556e11319f22ca9e3109c81df92))
+
+
+### Miscellaneous Chores
+
+* Bump all prod dependencies ([88cc76d](https://github.com/waku-org/js-waku/commit/88cc76d2b811e1fa4460207f38704ecfe18fb260))
+* Bump libp2p dependencies ([803ae7b](https://github.com/waku-org/js-waku/commit/803ae7bd8ed3de665026446c23cde90e7eba9d36))
+* Bump typescript ([12d86e6](https://github.com/waku-org/js-waku/commit/12d86e6abcc68e27c39ca86b4f0dc2b68cdd6000))
+
+
+### Dependencies
+
+* The following workspace dependencies were updated
+  * dependencies
+    * @waku/interfaces bumped from * to 0.0.8
+    * @waku/proto bumped from * to 0.0.3
+    * @waku/utils bumped from * to 0.0.2
```

Comment:
Does it only include workspace dependency? could it include all dependencies update by any chance?

Reply:
interesting, we may want to track some libp2p deps and typescript but fine to leave it as is for now.

Reply:
interesting, we may want to track some libp2p deps and typescript but fine to leave it as is for now.

Reply:
@fryorcraken why would we want to keep track of `libp2p` in our repo? It's just a dependency with some particular version.

Reply:
Because Waku is compatible only for a given version of libp2p which is a **peer** dependency of `@waku/core` (and full fledged dependency of `@waku/create`) and hence library consumers should be informed of its bump if they were to use `@waku/core` without `@waku/create` (which is not possible right now but something we want to achieve).

Reply:
In that case I believe it is enough to use [package.json](https://github.com/waku-org/js-waku/blob/8d8874d2bbd99979313581b498e44f41dbba0ac8/packages/core/package.json#L134) as it is now. Given that when we will change `libp2p` version it will be shown in changelog. Based on that I think there is no necessity to track `libp2p` separately. Any updates caused by usage of new version of it will be reflected as a commit and version bump.